paasta_tools/api/views/instance.py

```diff
@@ -113,3 +116,54 @@ def instance_status(request):
         raise ApiFailure(error_message, 500)
 
     return instance_status
+
+
+@view_config(route_name='service.instance.tasks.task', request_method='GET', renderer='json')
+def instance_task(request):
+    status = instance_status(request)
+    task_id = request.swagger_data.get('task_id', None)
+    verbose = request.swagger_data.get('verbose', False)
+    try:
+        mstatus = status['marathon']
+    except KeyError:
```

Comment:
👍 this sounds good. As discussed, I'm going to tackle this in a separate branch and add a basic `paasta status` implementation too